### PR TITLE
Fix media browser and contact group create

### PR DESCRIPTION
**Pull request by tulimaki**

### CODE REVIEW

File: shuup/admin/modules/media/static_src/media/browser/components/listFileView.js [hunk 1]
```diff
@@ -6,13 +6,12 @@
  * This source code is licensed under the OSL-3.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-const _ = require("lodash");
-const m = require("mithril");
-const moment = require("moment");
-const wrapFileLink = require("./wrapFileLink");
-const folderLink = require("./folderLink");
-const menuManager = require("../util/menuManager");
```
Comment (chessbr) [inline shuup/admin/modules/media/static_src/media/browser/components/listFileView.js]: this guy is missing
image

File: shuup/admin/modules/media/static_src/media/browser/menus/fileContextMenu.js [hunk 2]
```diff
@@ -6,8 +6,7 @@
  * This source code is licensed under the OSL-3.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-const fileActions = require("../actions/fileActions");
```
Comment (chessbr) [inline shuup/admin/modules/media/static_src/media/browser/menus/fileContextMenu.js]: this is missing
image

File: shuup/admin/modules/media/static_src/media/browser/menus/folderContextMenu.js [hunk 3]
```diff
@@ -7,8 +7,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 /* eslint-disable no-bitwise */
-const folderActions = require("../actions/folderActions");
```
Comment (chessbr) [inline shuup/admin/modules/media/static_src/media/browser/menus/folderContextMenu.js]: this guy is missing

**Comment by tulimaki** [comment]
Fixed!

**Pull request merged**